api/webscraper/nyserda_scraper.py

```diff
@@ -0,0 +1,98 @@
+import requests
+import json
+import pandas as pd
+from io import BytesIO
+from utils.scraper_utils import check_status, geocode_lat_long
+
+'''
+This scrapes data from the NYSERDA Large-scale Renewable Projects database.
+We filter for specific columns from the database's API and save them to a json file.
+https://data.ny.gov/Energy-Environment/Large-scale-Renewable-Projects-Reported-by-NYSERDA/dprp-55ye/about_data
+'''
+nyserda_large_response = requests.get('https://data.ny.gov/resource/dprp-55ye.json')
+if nyserda_large_response.status_code != 200:
+  raise ValueError('Request to NYSERDA failed. Status code: %d\n%s' % (nyserda_large_response.status_code, nyserda_large_response.text))
+else:
+  large_data = nyserda_large_response.json()
+  filtered_list = []
+  for item in large_data:
+    if check_status(item.get('project_status', None)) != 'Cancelled':
+      project_dict = {'project_name': item.get('project_name', None),
+                      'project_status': check_status(item.get('project_status', None)), 
+                      'renewable_energy_technology': item.get('renewable_technology', None),
+                      'size': item.get('bid_quantity_mwh', None),
+                      'developer': item.get('developer', None),
+                      'proposed_cod': item.get('year_of_delivery_start_date', None),
+                      'county': item.get('county_province', None),
+                      'region': item.get('redc', None),
+                      'zipcode:': item.get('zip_code', None),
```

Comment:
nit: should be `'zipcode':`

Reply:
@nehaahussain ah yes! it's a little funky but that's the way they titled the "zipcode" field for the NYSERDA large-scale data